features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/EmptyReportScenario.kt

```diff
@@ -3,31 +3,22 @@ package com.bugsnag.android.mazerunner.scenarios
 import android.content.Context
 import com.bugsnag.android.Bugsnag
 import com.bugsnag.android.Configuration
-import java.io.File
 
 internal class EmptyReportScenario(
     config: Configuration,
     context: Context,
     eventMetadata: String
 ) : Scenario(config, context, eventMetadata) {
 
-    init {
-        config.autoTrackSessions = false
-        val errDir = File(context.cacheDir, "bugsnag-errors")
-
-        if (eventMetadata != "non-crashy") {
+    override fun startBugsnag(startBugsnagOnly: Boolean) {
+        if (startBugsnagOnly) {
             disableAllDelivery(config)
-        } else {
-            val files = errDir.listFiles()
-            files.forEach { it.writeText("") }
```

Comment:
This block needs reinstating for the scenario to be testing anything.

Reply:
Reinstated, within the new format.